### PR TITLE
Attack logs are no longer generated after roundend

### PR DESCRIPTION
**Pull request by F-OS**
This prevents adminspam.

~~This code, in it's entirety, is released under the CC-0 license, the author disclaims all liability, to the extent permitted under applicable law, and authorizes all use of this code for any purposes.~~

~~Upon merging this code of course falls under the repo license, however within this PR i'm keeping it under CC-0~~

This was a meme

### CODE REVIEW

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -26,6 +26,8 @@ var/global/floorIsLava = 0
 			to_chat(C, msg)
 
 /proc/msg_admin_attack(var/text) //Toggleable Attack Messages
+	if(ticker.current_state == GAME_STATE_FINISHED)
+		return 0
```
Comment (Ghommie) [inline code/modules/admin/admin.dm]: return FALSE

File: code/modules/admin/admin.dm [hunk 2]
```diff
@@ -35,8 +37,11 @@ var/global/floorIsLava = 0
 				to_chat(C, msg)
 
 /proc/msg_admin_ff(var/text)
+	if(ticker.current_state == GAME_STATE_FINISHED)
+		return 0
```
Comment (Ghommie) [inline code/modules/admin/admin.dm]: return FALSE

**Comment by Ghommie** [comment]
Beat me to it by seconds, F-OS.

**Comment by Supermichael777** [comment]
Your code must be AGPL, this is non negotiable

**Pull request merged**